sklearn/svm/classes.py

```diff
@@ -177,6 +177,11 @@ def fit(self, X, y):
         y : array-like, shape = [n_samples]
             Target vector relative to X
 
+        sample_weight : array-like, shape = [n_samples], optional
+                        Array of weights that are assigned to individual
```

Comment:
Keep the same alignment as in the other docstrings: all the lines after the parameter first line are indented by four spaces.